src/main/java/com/squareup/kotlinpoet/AnnotationSpec.kt

```diff
@@ -15,6 +15,7 @@
  */
 package com.squareup.kotlinpoet
 
+import com.squareup.kotlinpoet.jvm.asClassName
```

Comment:
These imports will disappear when the factory methods that use them are moved into "jvm"